### ISSUE TITLE
JTS OverlayNG for tiling?

### ISSUE THREAD

**Issue by dr-jts** [jericks/geoc#5]
The new JTS [OverlayNG](http://lin-ear-th-inking.blogspot.com/2020/05/jts-overlay-next-generation.html) should be a lot better for generating data for vector tiles (it provides both  optimized rectangle clipping and precision reduction to the tile coordinate system).   Would this be useful for geoc?

**Comment by jericks** [comment]
I am really excited about trying out OverlayNG.  It will be really useful for geoc, and also the geometry commands(https://github.com/jericks/geometrycommands) and geoscript (https://github.com/geoscript/geoscript-groovy) projects I work on.  Would it be helpful if I tested these projects with your new branch?

**Comment by dr-jts** [comment]
> Would it be helpful if I tested these projects with your new branch?

Yes, absolutely.  Branch is [here](https://github.com/dr-jts/jts/tree/overlay-sr).  Check out these TestBuilder [functions ](https://github.com/dr-jts/jts/blob/overlay-sr/modules/app/src/main/java/org/locationtech/jtstest/function/OverlayNGFunctions.java)for entry points.
